app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def file_creator():
+    name_of_file = input("Enter name of the file: ")
+    with open(f'{name_of_file}.txt', 'a') as f:
```

Comment:
better use 'w' mode, as your task is to create a new file, not to add something to the existing one